doc/source/conf.py

```diff
@@ -15,11 +15,10 @@
 
 import sys
 import time
+from importlib.metadata import version
 from pathlib import Path
 
-# Attempt to import the version dynamically from GitHub tag.
 try:
-    from importlib.metadata import version
 
     fullversion = version("scikit_package")
```

Comment:
Unlike our existing GitHub CI for doc building, in readthedocs, it simply installs `docs.txt` only (no conda.txt) so there is no "metadata" available for this PR deployed version.

Reply:
> I wonder if a better solution is to add importlib to docs.txt just in this repo? 

`importlib` is the default module in Python so I don't think we have to import this.

> Do we not think that people may want to know the version that is running? I guess this doesn't work in principle because it just pulls the latest version of v0 and we don't do formal releases.

We aren't using our release-script centralized workflow but rather we are using the CI provided by `readthedocs` like this https://app.readthedocs.org/projects/scikit-package/builds/28161659/ I will see if we can grab the tag dynamically and I will create an issue.